db/schema.rb

```diff
@@ -88,6 +88,19 @@
   add_index "bundle_products", ["bundle_product_id"], name: "fk_bundle_prod_prod", using: :btree
   add_index "bundle_products", ["product_id"], name: "fk_bundle_prod_bundle", using: :btree
 
+  create_table "card_readers", force: :cascade do |t|
+    t.integer  "control_device_id", limit: 4, null: false
+    t.boolean  "entrance",                    null: false
+    t.datetime "created_at",                  null: false
+    t.datetime "updated_at",                  null: false
+  end
+
+  create_table "control_devices", force: :cascade do |t|
+    t.integer  "secure_room_id", limit: 4, null: false
```

Comment:
I'd maybe like this to be genericized to `product_id`. I could go either way, though.

But we should add an index and a foreign key on it (fyi, mysql automatically creates an index when you create a foreign key, but oracle doesn't, so you'll need to explicitly `add_index` as well as adding the foreign key.